src/store/actions.ts

```diff
@@ -1042,6 +1042,10 @@ export const startAnalyzeName: ActionIF = async ({ commit, getters }) => {
       commit('setErrors', 'length')
       return
     }
+    if (getters.getName.length > 150) {
+      commit('setErrors', 'long')
```

Comment:
Please be more explicit and call it 'mras_length_exceeded'

Reply:
Also, create a constant for those max and min values.

Reply:
done.